packages/overlays/src/LocalOverlayController.js

```diff
@@ -71,8 +99,8 @@ export class LocalOverlayController {
    * @param {boolean} [options.isShown] whether the overlay should be shown
    * @param {object} [options.data] overlay data to pass to the content template function
    */
-  sync({ isShown, data } = {}) {
-    this._createOrUpdateOverlay(isShown, data);
+  async sync({ isShown, data } = {}) {
```

Comment:
So yeah, we have "async sync" now 🙈  but it seems necessary to allow user to do a sync where they want to make sure that positioning (using popper, which is async since loading the module is lazy loaded) has finished updating...